language/diem-tools/writeset-transaction-generator/src/release.rs

```diff
@@ -71,3 +81,95 @@ pub fn create_release_writeset(
         vec![],
     )))
 }
+
+pub fn verify_payload_change<I: DiemValidatorInterface>(
```

Comment:
Nit: comment (can probably just lift some of the already-good code comments to doc ones).